models/image_classification/model_templates/custom-classification/README.md

```diff
@@ -190,3 +190,29 @@ python export.py \
 
 This produces model `model.xml` and weights `model.bin` in single-precision floating-point format
 (FP32). The obtained model expects **normalized image** in planar BGR format.
+
+
+### 8. Optimization
+
+The models can be optimized -- compressed by [NNCF](https://github.com/openvinotoolkit/nncf) framework.
+
+At the moment, only one compression methods is supported for image classification models: int8 quantization.
+
+Please, note that NNCF framework requires a dataset for compression, since it makes several steps of fine-tuning after
```

Comment:
Done

Reply:
I still believe this comment wasn't addressed. The following text is confusing: 
_Please, note that NNCF framework requires a dataset for compression, since it makes several steps of fine-tuning after
compression to restore the quality of the model, so the command line parameters of the script `compress.py` are closer
to the command line parameter of the training script for fine-tuning scenario 4.c stated above, but the number of epochs
required for NNCF compression should not be set by command line parameter, since it is calculated by the script
`compress.py` itself_
Why do we need to say it at all? Can we simply say that the compress script takes the same parameters as the train script?

Reply:
Maxim, we could say that, but I'm not sure that this will make the documentation better. 

In my opinion, if a person did not work with compression algorithms, then his/her "default" understanding of compression algorithm is that they do not require training at all, so the dataset is not required. 
(At least this was my understanding before I started work with NNCF/POT)
In this case it will not be clear (for such people) why the compress script's command line parameters likes to the command line parameters of the training script.
It will look like we hide some important details, whereas it is easy to describe them explicitly.

I will make some re-wording of this paragraph to make it shorter, but I still suppose that it is better to keep it.

Reply:
I don't propose to remove the paragraph. I propose to make it more concrete and avoid confusing phrases like "the command line parameters of the script compress.py are closer
to the command line parameter of the training script for fine-tuning scenario". Can we say exactly which parameters should be taken by the user from the training/finetuning?

Reply:
Maxim, it is possible to add more information on parameters for compression, but I have to say that at the moment all the other sections (how to start training, finetuning, how to make export, how to evaluate models) do not contain description of the parameters at all -- only examples how to call the scripts.
So, I just made the description for `compress.py` be similar to descriptions of OTE scripts in other sections (or tried to make it a bit more detailed)

Probably I chose not best wording in this section, -- please, correct it.

Unfortunately I'm in vacation from July 26 and will not be able to be online from today.
But if you (or somebody other) propose changes in the documentation, Ilya Krylov will be able to accept them and merge this PR.